k8s/helmfile/helmfile.yaml

```diff
@@ -198,9 +203,23 @@ releases:
     chart: ./../../charts/stats-cron
     <<: *default_release
 
+  - name: prometheus-elasticsearch-exporter
```

Comment:
This also wants to only be installed in the local env for now; at the moment this also adds the exporter to all environments

Reply:
Isn't that what we want?

Reply:
We were historically favouring PRs touching the smallest number of environments at a time.

The reason being that if, or example, this fails to apply to production (e.g for some complicated reason due to CRDs [which I think it actually will because the prometheus-operator.yaml *isn't* being applied to production]) then we end up reverting this whole PR (or blocking other changes being made to cluster)

Of course "in the fullness of time" (probably very soon) we do want this in both staging and production